engine/src/main/java/org/hibernate/validator/internal/engine/messageinterpolation/el/SimpleELContext.java

```diff
@@ -6,74 +6,52 @@
  */
 package org.hibernate.validator.internal.engine.messageinterpolation.el;
 
-import java.lang.reflect.Method;
-
 import javax.el.ArrayELResolver;
 import javax.el.BeanELResolver;
 import javax.el.CompositeELResolver;
-import javax.el.ELContext;
 import javax.el.ELResolver;
 import javax.el.ExpressionFactory;
 import javax.el.ListELResolver;
 import javax.el.MapELResolver;
 import javax.el.ResourceBundleELResolver;
-import javax.el.ValueExpression;
-import javax.el.VariableMapper;
+import javax.el.StandardELContext;
+import javax.el.StaticFieldELResolver;
 
 /**
  * @author Hardy Ferentschik
+ * @author Guillaume Smet
  */
-public class SimpleELContext extends ELContext {
+public class SimpleELContext extends StandardELContext {
```

Comment:
There are 2 reasons why I kept our specific context:
- first, we provide a way to inject a specific `ExpressionFactory` and believe it or not, the one we passed to `StandardELContext` is not used, except to initialize the `StreamELResolver` (see the constructor of `StandardELContext`). See ELContext#convertToType to see how it defaults to the static one if it is not found in the context. So if we want to be sure it's used, we need to put it in the context.
- the second reason is that, with a `StandardELContext`, we instantiate new resolvers for each interpolation, while they are stateless and can be shared.

Note that to keep the list of resolvers static, I didn't add the `StreamELResolver` which didn't look as much value to me (and wasn't present before). However, I added the `StaticFieldELResolver`.